src/client/src/ui/regions/views/workspace/workspaceRegionView.jsx

```diff
@@ -38,6 +38,6 @@ export default class WorkspaceRegionView extends ViewBase {
           {View}
         </WorkspaceItemContainer>
       );
-    });
+    }).concat(_.times(3, () => <div className='workspace-region__spacer'></div>)); // add empty items at the end so tiles lay out nicely
```

Comment:
Yeah, this is a bit of magic. This is so all the tiles have the same width, even when a row is not full, but yet, tiles can stretch.